public/script.js

```diff
@@ -1,5 +1,38 @@
-const earthDateParagraph = document.querySelector("#earth-date");
 const earthDate = new Date();
-const dateText = document.createTextNode(earthDate.toLocaleString());
 
-earthDateParagraph.appendChild(dateText);
+function displayCurrentEarthDate() {
+  const dateText = document.createTextNode(earthDate.toLocaleString());
+  document.querySelector("#earth-date").appendChild(dateText);
```

Comment:
I often find it's a bit more readable if the extra chains are on separate lines (and aligned nicely)

```
    document.querySelector("#earth-date")
            .appendChild(dateText);
```

(And same for the rest of the chaining in this file)

Reply:
My Prettier VS Code extension (which formats on save) doesn't allow me to change the first two instances of the chaining. I am planning to setup Prettier and ESLint to run as a commit hook in this repo in the future though (and I will add that rule to move chains onto separate lines when I set that up) do you think I should do this as part of this PR, or as a separate PR? 🤔